src/common/crypto.ts

```diff
@@ -0,0 +1,16 @@
+import utils from 'keystore-idb/utils'
+
+export const sha256 = async (buf: ArrayBuffer): Promise<ArrayBuffer> => {
+  return window.crypto.subtle.digest(
+    {
+        name: "SHA-256",
+    },
+    buf
+  )
+}
+
+export const sha256Str = async(str: string): Promise<string> => {
+  const buf = utils.strToArrBuf(str, 8)
```

Comment:
Should we create `utils.strToUtf8Buf(str)` and `utils.strToUtf16Buf(str)` instead? Might be more helpful?

Reply:
So:
- `utils.strToUtf8Buf(str) = utils.strToArrBuf(str, 8)` and
- `utils.strToUtf16Buf(str) = utils.strToArrBuf(str, 16)`?

Sure! I don't see a huge difference, but it might read a bit better

Reply:
Yeah that'd be great. My reasoning is that, this way you can't use any `charSize` that doesn't make sense 🤷

Reply:
yeah that makes sense to me